src/admin/class-stats.php

```
@@ -84,6 +84,25 @@ private static function compute_stats( $existing_stats = null ) {
 					}
 
 					$cimo_data = $metadata['cimo'];
+
+					/**
+					 * This data comes from bulk optimizing media in the Media Library:
+					 */
+
+					if ( isset( $cimo_data['bulk_optimization'] ) ) {
+						foreach ( $cimo_data['bulk_optimization'] as $size => $data ) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$size'. (undefined)

(UnusedLocalVariable)